packages/sitecore-jss-react/package.json

```diff
@@ -27,7 +27,7 @@
     "url": "https://github.com/sitecore/jss/issues"
   },
   "devDependencies": {
-    "@sitecore-feaas/clientside": "^0.4.4",
+    "@sitecore-feaas/clientside": "0.4.12",
```

Comment:
Why the exact version here? Should at least match our peerDependencies (~). 
In general though, are we anticipating breaking changes in @sitecore-feaas/clientside minor versions? If following semver (which we should be), then ^ should be fine.

Reply:
yeah needs ^ probably, if it was copied from my app - sorry

Reply:
> In general though, are we anticipating breaking changes in @sitecore-feaas/clientside minor versions

We don't, but sometimes they happen. I've encountered two instances during Forms integration dev. I can bring the ^ back if we don't expect any other ones.
@ambrauer

Reply:
Updated the version